package.json

```diff
@@ -105,7 +106,7 @@
     "nypr-account-settings": ">=0.2.2",
     "nypr-ads": ">=0.1.0",
     "nypr-audio-services": "^0.5.6",
-    "nypr-auth": "nypublicradio/nypr-auth#7642c031a845d45f251c7c1ab2c45e47892f4fa0",
+    "nypr-auth": "nypublicradio/nypr-auth#479530fff261b0dbf95dcdc800a9c236a97e7200",
```

Comment:
maybe better to just point to the branch (`nypublicradio/nypr-auth#bscharenberg/wqxr-on-fastboot`) rather than the specific commit so it'll pick up the head if you need more changes? or should we merge your work there and bump the version on `nypr-auth` real quick and just use `0.2.4`? (or `0.3.0`?)

Reply:
That's a good point.. I will update to use the wqxr-on-fastboot branch. I'm going to hold off on merging that work, since we are planning on a fastboot cleanup ticket. And the scope of that ticket will be a number of things, one of which will be to make sure all the nypr-libraries are using the published versions again!

Reply:
Fixed!